src/XmlNotepad/FormMain.cs

```diff
@@ -205,7 +205,6 @@ public class FormMain : System.Windows.Forms.Form, ISite {
         private ToolStripMenuItem changeToCDATAContextMenuItem;
         private ToolStripMenuItem changeToCommentContextMenuItem;
```

Comment:
Thanks, but I'd prefer the diff options to be integrated into the Options dialog, I'll be happy to do that work if you are too busy, let me know, thanks.

![Untitled](https://user-images.githubusercontent.com/18707114/58683993-6dc57400-832b-11e9-846c-5594a4edc39c.png)

Reply:
My reasoning was that some users might want to frequently change the options after viewing the results of their 1st diff. When you have too many differences you often want to disable some options to redo a more focused diff. Or maybe the diff-options were not valid for a particular set of XML files.

I considered adding it to the Options dialog, but for me global options are user preferences that will persist across many sessions, or are at least not frequently changed. Conceptually I wanted to accommodate that the diff-options can potentially change between every diff, even though I concede that, for most users, it probably won't.

I considered opening a dialog when you trigger the diff that shows the currently selected options, which you can then change, but decided against it since it might get irritating if for those users who never need to change their options.

That was why I thought it could be useful to have the diff-options separate but close to the diff 'trigger'.

OTOH, an option I did not consider which may even be better, is to add it to Options as you say, but then to show a Dialog with the specified diff-options when a diff is triggered. The diff-options can then still be changed (which also changes the diff-options under Options to persist them). But this dialog can have a "Do not show again" tick-box, which would not show the dialog on subsequent diffs defaulting to the diff-options specified in Options. 

What do you think of this idea?

Reply:
Sounds to me what we really want then is for the user to select "compare files" select the second file, but then the resulting window that opens has additional options to specify the type of diff you want (a wrapper around the web browser) so you can "fiddle" with the diff output until you get what you want.  This window then re-runs the diff as many times as necessary to show the results "live" as you change the options in a top level toolbar on that window...

Reply:
while we work on the design of this new dialog, I merged your fixes and added the settings and options dialog UI for XmlDiffOptions.

Reply:
Agreed on your proposal - that sounds even better. 

OK, I'll pull your changes and have a look. Sounds promising anyway. (Sorry for the delay in responding - it was a long weekend in France and I was away).

Thanks for being so responsive on this.